bin/improver-wind-downscaling

```diff
@@ -0,0 +1,104 @@
+#   list of conditions and the following disclaimer.
+#
+# * Redistributions in binary form must reproduce the above copyright notice,
+#   this list of conditions and the following disclaimer in the documentation
+#   and/or other materials provided with the distribution.
+#
+# * Neither the name of the copyright holder nor the names of its
+#   contributors may be used to endorse or promote products derived from
+#   this software without specific prior written permission.
+#
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+# AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+# IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+# ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE
+# LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
+# CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+# SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+# INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+# CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+# ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+# POSSIBILITY OF SUCH DAMAGE.
+"""Script to run wind downscaling."""
+
+import argparse
+
+import iris
+
+from improver import wind_downscaling
+
+
+def main():
+    """Load in arguments and get going."""
+    parser = argparse.ArgumentParser(
+        description='Run wind downscaling to apply roughness correction and'
+                    ' height correction to wind fields (as described in'
+                    ' Howard and Clark [2007]).')
+    parser.add_argument('wind_speed_filepath', metavar='WIND_SPEED_FILE',
+                        help='Location of the wind speed on standard grid'
+                             ' file. Any units can be supplied.')
+    parser.add_argument('silhouette_roughness_filepath', metavar='AOS_FILE',
+                        help='Location of model silhouette roughness file. '
+                             'Units of field: dimensionless')
+    parser.add_argument('sigma_filepath', metavar='SIGMA_FILE',
+                        help='Location of standard deviation of model '
+                             'orography height file. Units of field: m')
+    parser.add_argument('target_orog_filepath',
+                        metavar='TARGET_OROGRAPHY_FILE',
+                        help='Location of target orography file to downscale'
+                             ' fields to.'
+                             'Units of field: m')
+    parser.add_argument('standard_orog_filepath',
+                        metavar='STANDARD_OROGRAPHY_FILE',
+                        help='Location of orography on standard grid file '
+                             '(interpolated model orography.'
+                             ' Units of field: m')
+    parser.add_argument('model_resolution', metavar='MODEL_RESOLUTION',
```

Comment:
Just a question, how does this deal with the variable resolution over the model domain?